docs/develop/api-reference/auth-tokens.md

```diff
@@ -0,0 +1,174 @@
+
+For cache, the value can be the built-in `AllCaches` or a string value containing the name of the cache this permission is for.
+
+### TopicPermission
+A component of a [TokenScope](#tokenscope) object that defines permissions for a token.
+
+| Name            | Type                              | Description                                                                          |
+| --------------- |-----------------------------------|--------------------------------------------------------------------------------------|
+| role           | SubscribeOnly \| PublishSubscribe | The type of access granted by the permission.               |
+| cache          | `AllCache` \| String              | A permission can be restricted to a select cache by name or AllCaches built-in value.              |
+| topic          | `AllCache` \| String              | A permission can be restricted to a select topic by name or AllTopics built-in value. |
+
+For role, there are two managed roles to assign, `TopicRole.PublishSubscribe` and `TopicRole.SubscribeOnly`. Custom roles are not supported. Using the SubscribeOnly role allows only subscribing to topics, whereas using the PublishSubscribe role allows publishing and subscribing to topics.
+
+For cache, only topics within that cache's namespace are allowed by the permission. This can be set to the built-in `AllCache` value or a string specifically naming a cache.
+
+For topic, this can be set to the built-in `AllTopics` value, which gives access to all topics in the cache(s) defined in cache or it can be a string with a specific topic name.
+
+#### TopicScope examples
+This is an example of creating a TokenScope with just CachePermissions.
+
+```javascript
+const CachePermissions = {
+    permissions: [
+        {
+            role: CacheRole.ReadWrite, // Managed role
+            cache: "MyCache" // grants access to a specific cache
+        },
+        {
+            role: CacheRole.ReadOnly, // Managed role
+            cache: AllCaches // Built-in value for access to all caches in the account.
+        },
+    ],
+};
+```
+
+This is an example of creating a TokenScope with just TopicPermissions.
+
+```javascript
+const TopicsPermissions = {
+    permissions: [
+      {
+        role: TopicRole.PublishSubscribe, // Managed role
+        cache: 'the-great-wall', // grants access to a specific cache
+        topic: 'highlights', // grants access to a specific topic
+      },
+      {
+        role: TopicRole.SubscribeOnly, // Built-in role
+        cache: AllCaches, // This is a built-in value for access to all caches in the account
+        topic: AllTopics, // This is a built-in value for access to all topic in the listed cache(s).
+      },
+    ],
+};
+```
+
+:::tip
+
+If you have any questions not answered here, please jump on [our Discord server](https://discord.gg/2c24SK6W) and ask our experts in the support channel.
+
+:::
+
+## FAQ
+
+<details>
```

Comment:
Another question worthwhile adding is that we do now allow tokens with write only or publish only permissions

Reply:
I will add this.